### ISSUE TITLE
On Install Page

### ISSUE THREAD

**Issue by bvelitchkine** [Bastien-and-Gauvain/monorepo#71]
Create the page to display as soon as the extension is installed (cf. [developers doc](https://developer.chrome.com/docs/extensions/reference/runtime/#event-onInstalled)). It should look like this:

![Image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/1df00355-7423-435f-bdbb-42af1cd67b76)

And it should have a developer-friendly API to modify the UI depending on the step the user reached in the onboarding. There should also be a global "onboarding" state variable.

**Comment by GauvainThery** [comment]
~Blocked by #70~

**Comment by GauvainThery** [comment]
This would require to add a new column on the user table called onboarding_state, an enum 'NOT_STARTED', 'CONNECTED_TO_NOTION', 'FIRST_PROFILE_SAVED', 'EXTENSION_PINNED' that is nullable

**Comment by GauvainThery** [comment]
How I decided to work on this:

- [x] Frontend integration of the page as a tab page (cf. [plasmo](https://docs.plasmo.com/framework/tab-pages))
- [ ] Add background logic (ie. resolver) to get user (should be already done)
- [ ] And to update users based on this 3 behaviours: (first) sign in with Notion, first contact saved, extension pinned
- [ ] Add the trigger to those events where they should be
- [ ] Update the page to display dynamically the elements based on the current onboarding of the user
- [ ] Make sure to open this page onInstall (when the tasks are not all completed only, as onInstalled can be fired on update of the extension and of chrome)
